alf/networks/param_networks.py

```diff
@@ -183,20 +193,26 @@ def __init__(self,
                 tuple takes a format
                 ``(filters, kernel_size, strides, padding, pooling_kernel)``,
                 where ``padding`` and ``pooling_kernel`` are optional.
-            fc_layer_params (tuple[tuple]): a tuple of tuples where each tuple
-                takes a format ``(FC layer sizes. use_bias)``, where
-                ``use_bias`` is optional.
+            fc_layer_params (tuple[int]): a tuple of integers
```

Comment:
Perhaps a more generalizable way of doing this is to implement ParamSequential

Reply:
Yeah, that would be ideal. I will think it over after the submission deadline.